site/sfguides/src/dora/snowflake-auto-grader.md

```diff
@@ -99,16 +98,69 @@ select grader(step, (actual = expected), actual, expected, description) as grade
  'DORA_IS_WORKING' as step
```

Comment:
Does this need to change too? The name of the step here

Reply:
yes